datadog-setup.php

```diff
@@ -308,12 +319,6 @@ function install($options)
 
             // phpcs:disable Generic.Files.LineLength.TooLong
             if ($shouldInstallAppsec) {
-                // Appsec crashes with missing symbols if tracing is not loaded
-                execute_or_exit(
-                    'Impossible to update the INI settings file.',
-                    "sed -i 's@ \?; \?extension \?= \?ddtrace.so@extension = ddtrace.so@g' "
-                        . escapeshellarg($iniFilePath)
-                );
```

Comment:
It looks like this can not be the case (missing `ddtrace.so` in `$iniFilePath`), as running `datadog-setup.php` will always install `ddtrace` extension and if it fails to do so, it will not reach this point, correct?

Reply:
Correct. Just previously, for some reason it did not uncomment the line if present, which was not expected and making the test fail under arm where appsec is not loaded :-)